src/services/ProductsService.ts

```diff
@@ -0,0 +1,117 @@
+import fs from "fs-extra"
```

Comment:
It will be better if we separate the imports like this:

1st put the imports that are from node
then put the lib imports (here you don't have any but anyway)
then put the imports that are like types, controllers, routes.

and separate each group with new line